core/frontend/src/render/webgl/RealityMesh.ts

```diff
@@ -376,7 +376,7 @@ export class RealityMeshGeometry extends IndexedGeometry implements IDisposable,
       branch.add(system.createBatch(primitive!, featureTable, mesh.getRange(), { tileId }));
     }
 
-    return system.createBranch(branch, realityMesh._transform ? realityMesh._transform : Transform.createIdentity());
+    return system.createBranch(branch, realityMesh._transform ? realityMesh._transform : Transform.createIdentity(), {overrideClipStyle: true});
```

Comment:
To fix this, a `disableClipStyle` boolean has been added to `RealityMeshParams`, and it is now being set in `MapTile.ts`, so `RealityMesh.ts` now only passes that information along instead of dictating it.